frontend/src/v4/modules/sequences/sequences.sagas.ts

```diff
@@ -277,6 +279,10 @@ function* handleTransparenciesVisibility({ transparencies }) {
 	}
 }
 
+export function* clearColorOverrides() {
```

Comment:
Is there any point having `clearColorOverrides` instead of just using `setColorOverrides(false)`?